src/webgui/src/webgui/server.py

```diff
@@ -1,12 +1,18 @@
 #!/usr/bin/env python
-from flask import Flask, render_template
+from flask import Flask, render_template, Response
 import rospy
 from std_msgs.msg import String
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
 import os
 import sys
 import rospkg
 import threading
+import cv2
 
+l_cam_topic = 'l_cam_topic'
```

Comment:
These are another good place to get parameters from rosparam

Reply:
`l_cam_topic = rospy.get_param('l_cam_topic')` should work right?

Reply:
Yup